### DISCUSSION TITLE
Add more loglikelihoods (ARMA variants and others)

### DISCUSSION THREAD

**Discussion by ben18785** [pints-team/pints#1580] [Ideas/Science]
Additive ARMA(1, 1) errors are just one of a slew of potential error models to consider including. Others include:

- [ ] Heteroscedastic errors where errors scale with value of y
- [ ] Multiplicative independent and ARMA errors
- [ ] ARMA processes using Student-t errors
- [ ] Higher order ARMA(p, q) processes
- [ ] Autoregressive-conditional-heteroscedacity (ARCH) and GARCH (G for general) processes, which allow an error process which changes with time
- [ ] Kalman filter state-space models (probably a whole student project there...)

In principle, #661 should highlight which, if any, of these processes is appropriate for data.

**Comment by MichaelClerx** [comment]
@ben18785 @rcw5890 is this still something we want to do?
